contracts/FungibleToken-v2-ContractInterface.cdc

```diff
@@ -0,0 +1,14 @@
+import FungibleToken from "./FungibleToken-v2.cdc"
+
+pub contract interface FungibleTokenInterface {
```

Comment:
Just adding a note here, maybe we should also have:

```
// is this type an accepted token?
pub fun isAcceptedType(type: Type): Bool

// I could also know if this type is accepted by asking for VaultInfo and 
// return an optional if it isn't?
pub fun getTokenType(type: Type): FungibleToken.VaultInfo?
```

Reply:
It is already possible to get this with the `getVaultTypes` function, so is it necessary to have these also?

Reply:
Everything these would do are doable, these were just because previously `getVaultTypes` was returning an array. It could probably be argued that `isAcceptedType` could still be kept but it really would just be a wrapper around `getVaultTypes` so it might just be bloat and better off left out